internal/ui/styles.go

```
@@ -1,34 +1,35 @@
+	colorOverlay color.Color
+	colorBorder  color.Color
+	colorBright  color.Color
+	colorText    color.Color
+	colorDim     color.Color
+	colorSubtle  color.Color
+	colorAccent  color.Color
+	colorAccent2 color.Color
+	colorSelBg   color.Color
+
+	colorWorking  color.Color
+	colorWaiting  color.Color
```

golangci-lint (2.13.2)

[error] var colorWaiting is unused

(unused)